src/vivarium_nih_us_cvd/components/risks.py

```diff
@@ -20,3 +28,70 @@ def _get_current_exposure(self, index: pd.Index) -> pd.Series:
             exposures[exposures < min_exposure] = min_exposure
             exposures[exposures > max_exposure] = max_exposure
         return exposures
+
+
+class SBPRisk(Risk_):
+    """Manages gbd SBP exposure and untreated SBP exposure pipelines"""
+
+    def __init__(self, risk: str):
+        super().__init__(risk)
+        self.gbd_exposure_pipeline_name = f"{self.risk.name}.gbd_exposure"
+
+    def __repr__(self) -> str:
+        return f"Risk({self.risk})"
+
+    #################
+    # Setup methods #
+    #################
+
+    # noinspection PyAttributeOutsideInit
+    def setup(self, builder: Builder) -> None:
+        self.gbd_exposure = self._get_gbd_exposure_pipeline(builder)
+        super().setup(builder)
```

Comment:
I think we established the order doesn't actually matter here, so I'd suggest switching it back as it looks nicer to have  the `super` call first. If it does in fact need to be in this order, disregard this comment.